content/api/_index.md

```diff
@@ -35,6 +35,19 @@ If this file is missing or incorrectly named, your robot won't do anything. No l
 ## Start Button
 After the robot has finished starting up, it will wait for the _Start Button_ on the power board to be pressed before continuing with your code, so that you can control when it starts moving. There is a green LED next to the start button which flashes when the robot is finished setting up and the start button can be pressed.
 
+### Doing things before pressing the start button
+If you want to do things before the start button press, such as setting servos or motors, you can pass `wait_for_start_button`. You will then need to wait for the start button [manually](power-board/#start-button).
```

Comment:
`such as setting servos or motors`  - I think this might be a typo, did you mean to write `setting up`?